test/tsconfig.json

```diff
@@ -44,6 +44,7 @@
         "./rules/indentRuleTests.ts",
         "./rules/interfaceNameRuleTests.ts",
         "./rules/jsdocFormatRuleTests.ts",
+        "./rules/keywordNamedVariablesRule.ts",
```

Comment:
Ah, [this](https://github.com/palantir/tslint/blob/master/test/tsconfig.json#L20) might explain it. Do you still need/want this explicit list of files?